pkgs/by-name/sa/satisfactorymodmanager/package.nix

```
@@ -78,9 +81,26 @@ buildGoModule rec {
   installPhase = ''
     runHook preInstall
     install -Dm755 build/bin/SatisfactoryModManager -t "$out/bin"
+
+    for i in 16 32 64 128 256 512; do
+      install -D ./icons/''${i}x$i.png $out/share/icons/hicolor/''${i}x$i/apps/SatisfactoryModManager.png
```

```suggestion
      install -D ./icons/"$i"x"$i".png "$out"/share/icons/hicolor/"$i"x"$i"/apps/SatisfactoryModManager.png
```

This is just personal preference